index.html

```diff
@@ -26,10 +26,21 @@
             </div>
         </a>
         <div class="desktop-nav">
```

Comment:
- [ ] Great first implementation!  However, the elements on the menu are not centered.   Please, center them so the menu looks exactly like the Figma.
- [ ] The first element "Home", is not consistent with the horizontal spacing of the other elements.  Check your margins/paddings/top:

![image](https://user-images.githubusercontent.com/57421823/162785235-a1722ca2-dde0-4b7f-9029-722877779673.png)

- [ ] Kindly make the menu disappear when an element is touched/clicked.   "Portfolio" and "Contact" do not have this required functionality.

Reply:
Hi @Meltrust! Thanks for the quick review. I have:
- Centered the menu. Please test with 320px screen width for alignment.
- Adjusted the horizontal spacing between the links.
- Fixed the links to the required functionality.